product-service/serverless.ts

```diff
@@ -0,0 +1,55 @@
+import { Serverless } from 'serverless/aws';
+
+const serverlessConfiguration: Serverless = {
+  service: {
+    name: 'service-product',
+  },
+  frameworkVersion: '2',
+  custom: {
+    webpack: {
+      webpackConfig: './webpack.config.js',
+      includeModules: true,
+    },
+  },
+  // Add the serverless-webpack plugin
+  plugins: ['serverless-webpack'],
+  provider: {
+    name: 'aws',
+    runtime: 'nodejs12.x',
+    region: 'eu-west-1',
+    apiGateway: {
+      minimumCompressionSize: 1024,
+    },
+    environment: {
+      AWS_NODEJS_CONNECTION_REUSE_ENABLED: '1',
+    },
+  },
+  functions: {
+    getProductsList: {
+      handler: 'handler.getProductsList',
```

Comment:
Just FYI. It's not necessary to keep handler.ts in root directory, you can specify path to lambda function itself, e.g. `handler: src/handlers/getProductsList.getProductsList`

Reply:
Got it.
Thanks